### ISSUE TITLE
MS Word 2016 fails to focus on initial document upon launch

### ISSUE THREAD

**Issue by PratikP1** [nvaccess/nvda#7176]
Testing environment: 
NVDA version next-14045,c75e7d19
Windows 10 pro Insider build 16193 

When launching MS word 2016 either from the start menu or from the task bar, NVDA fails to automatically focus on "document 1." Rather, NVDA shows that it's focused on the Word logo. See log snippet. This behavior has occurred with builds before NVDA 2017.1. I believed the problem may have been related to Insider builds. I've tried this on several machines running Windows and NVDA. I have also tried this on different builds of Word.

* Expected behavior: NVDA should automatically focus in the edit field of the open document.
* Actual behavior: NVDA requires the user to press alt+tab to focus on the launched window.

STR:
1. Press Windows logo to launch Start menu.
2. Type Word to let Windows find Word 2016 and press enter.
3. Same could be achieved if Word is pinned to the taskbar by pressing the Windows+number associated with the item.
[NVDA MS Word Not speaking at launch.txt](https://github.com/nvaccess/nvda/files/999536/NVDA.MS.Word.Not.speaking.at.launch.txt)

**Comment by PratikP1** [comment]
This problem no longer occurs with Windows 10 build 16226 and NVDA version next-14129,104d2ff9. The installed Word version is 16.0.8201.2102

**Comment by fernando-jose-silva** [comment]
I have this problem using windows 7 and office 2013.

**Comment by PratikP1** [comment]
I was a bit premature in my pronouncement earlier today. It appears that I’m still having the issue I described in the original report.

**Comment by ehollig** [comment]
This sounds like a duplicate of #5768
